src/core/core.helpers.js

```diff
@@ -723,7 +723,7 @@ module.exports = function(Chart) {
 	};
 	helpers.addEvent = function(node, eventType, method) {
 		if (node.addEventListener) {
-			node.addEventListener(eventType, method);
+			node.addEventListener(eventType, method, {passive: true});
```

Comment:
Will adding this parameter cause issues on browsers other than Chrome? We should confirm that this doesn't break IE11 compatibility.